### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by happy5214** [hed-standard/hed-javascript#11]
The readme has not been updated to include short/long conversion, and is generally hard to add to. We should rewrite it to make it read more like an API document.

**Comment by garrettmflynn** [comment]
Just to be clear about my own aims, I'm part of a Google Summer of Code project where we're intending to _write_ to BIDS files from the browser. The full proposal can be found [here](https://summerofcode.withgoogle.com/programs/2022/projects/WOkMdu9V). In our case, it will be useful to iteratively check whether a user-inputted HED string is valid or not—rather than check with a full call to validate.BIDS() or a related subfunction. 

Going through the prep for this project, it's apparent that [bids-validator](https://github.com/bids-standard/bids-validator) does not expose its methods for file-loading and, as such, I've implemented some preliminary file-loading tools that can parse a directory of BIDS files into a JavaScript object. Assuming that [bids-validator](https://github.com/bids-standard/bids-validator) will continue to keep the details of its file-parsing and error-checking away from the surface of its API, it'd be greatly beneficial to have a standalone validator for strings.

Of course, if this is outside of scope, I can always just maintain my own fork.

**Comment by VisLab** [comment]
You are correct that the documentation is very much out of date. The `hed-validator` has been recently refactored, and the main interface is through [`hedvalidator.validator.validateBidsDataset`](https://github.com/hed-standard/hed-javascript/blob/481d5e38af62c28639c7b15c38315e891cedb62a/validator/bids/validate.js#L20).

The [linking code in BIDS](https://github.com/bids-standard/bids-validator/blob/master/bids-validator/validators/events/hed.js) shows intended interface use. This code might be adapted for use in your application.

One of the difficulties with just validating at the string level is that many of the new features of HED cannot be checked without an entire events file and accompanying merged sidecar.

We apologize for letting the README get out of date and for the minute are only going to expose the validation at the dataset level. 

The new README explains what is checked at various levels. 

Your project is very exciting, and we hope that you will contact us as needed for support, but right now our primary focus is on full implementation of the HED-3G features including onset-offset checking and library schemas.

**Comment by garrettmflynn** [comment]
No worries! Thank you for the quick updates. I'll keep in touch as needed and post separate issues to propose additional features we might benefit from / stumble across in our efforts.

@VisLab Would you mind posting the direct link to the new README? It doesn't appear to be updated on the master branch.

**Comment by VisLab** [comment]
It is in PR#74.  Please feel free to make suggestions before we merge with the master branch.

**Comment by garrettmflynn** [comment]
Looks good to me!
